### ISSUE TITLE
Missing uma0

### ISSUE THREAD

**Issue by eureckou** [xyzz/gamecard-microsd#28]
I followed the tutorial on wololo to use sd2vita. I successfully installed it. However I can't find uma0 in vitashell. I am under the impression that upon activating the plugin, sd2vita will be ux0 and the sony memory card will be uma0. I have a vita phat with FW6.68 running h-encore. SD2vita is 200GB and the memory card has 16GB. I want to use the sd2vita for vita games and the memory card for PSP games using Adrenaline. Is my impression wrong or did I miss something? Thanks.

**Comment by itsminht** [comment]
Has this issue been fixed? It's working fine for me here

**Comment by eureckou** [comment]
Nope. I end up using a different plugin to assign uma0 = mc, ux0 = sd2vita.

**Comment by Urbs97** [comment]
You need a modded version of gamesd.
The original one doesnt mount the mem card.

**Comment by windsurfer1122** [comment]
Have a look at https://github.com/CelesteBlue-dev/PSVita-StorageMgr

**Comment by Urbs97** [comment]
i'm having problems with storagemgr so i wanted to revert back to gamesd only.

**Comment by windsurfer1122** [comment]
Can you give me a link to your issue report over at StorageMgr?

**Comment by Jeff-Smart** [comment]
Here is the diff for a version that mounts the vita Memory Card to uma0 nad the sd Card to ux0, i have simply adapted a former fork to the version 1.4:

54a55,58
> #define MOUNT_POINT_ID2 0xF00 //used uma0 ID
>
>
>
83a88,89
> static SceIoDevice uma_uma0_dev = { "uma0:", "exfatuma0", "sdstor0:xmc-lp-ign-userext", "sdstor0:xmc-lp-ign-userext", MOUNT_POINT_ID2 }; //For Vita MU
>
87a94,101
> static int exists(const char *path) {
>       int fd = ksceIoOpen(path, SCE_O_RDONLY, 0);
>       if (fd < 0)
>               return 0;
>       ksceIoClose(fd);
>       return 1;
> }
>
93a108,111
> static void io_mount(int id) {
>       ksceIoMount(id, NULL, 0, 0, 0, 0);
> }
>
123a142,158
> int shellKernelRedirectUma0() {
>       SceIoMountPoint *mount = sceIoFindMountPoint(MOUNT_POINT_ID2);
>       if (!mount) {
>               return -1;
>       }
>
>       if (mount->dev != &uma_uma0_dev && mount->dev2 != &uma_uma0_dev) {
>               ori_dev = mount->dev;
>               ori_dev2 = mount->dev2;
>       }
>
>       mount->dev = &uma_uma0_dev;
>       mount->dev2 = &uma_uma0_dev;
>
>       return 0;
> }
>
166a202,205
>       shellKernelRedirectUma0(); //Added uma0 mount was ux0 ie Vita MU
>       io_mount(MOUNT_POINT_ID2); //No need to remount since it's not mounted!
>
>
I jave added the compiled file.

[gamesd.zip](https://github.com/xyzz/gamecard-microsd/files/2773772/gamesd.zip)
